### DISCUSSION TITLE
Can I receive email when using Amazon SES

### DISCUSSION THREAD

**Discussion by saileshkush95** [anymail/django-anymail#416] [Q&A]
Can i receive email when i use ses, because i saw inbound message features in docs.

**Comment by medmunds** [comment]
@saileshkush95 yes, Anymail provides support for receiving email through Amazon SES. See Anymail's docs under [Amazon SES - *Inbound webhook*](https://anymail.dev/en/stable/esps/amazon_ses/#inbound-webhook).

Like everything with SES, the configuration can be a bit complicated.
